src/components/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const Header = () => {
+  return (
```

Comment:
Do you know you can use implicit returns? 

```js
const Header = () => (
  <div className="bob">...</div>
)

```

Reply:
yes you are right.